systemtests/bin/setup_download_tests.py

```diff
@@ -52,18 +68,17 @@ def iterate_through_symbols_files(
             # We only test for sym files currently
             continue
 
-        is_try = False
-
         if sym_filename.startswith("try/"):
+            if not try_symbols:
+                continue
```

Comment:
If this is a symbols file in the try bucket and we're not looking for those, this skips to the next filename to look at.